test/common/tmpdir.js

```diff
@@ -33,11 +28,7 @@ function rmSync(pathname) {
     }
   }
 
-  if (err === null) {
-    return;
-  }
-
-  throw err;
+  if (err) throw err;
```

Comment:
Per GH actions:
```suggestion
  assert.ifError(err);
```

Reply:
Hmm, this ends up throwing an `ERR_ASSERTION` with the original error embedded in it, instead of just the original error. I think we should throw only if `err` is strictly not `null`, like this:
```suggestion
  if (err !== null) throw err;
```

Reply:
tbh you could get rid of the `if` statement, at this point we know that `err` cannot be null.

Reply:
Ah, correct! Updated. :D